apps/server-mcp/src/common/version.ts

```
@@ -0,0 +1,71 @@
+		for (const packageJsonPath of possiblePaths) {
+			try {
+				const packageJsonContent = readFileSync(packageJsonPath, 'utf8');
+				const packageJson = JSON.parse(packageJsonContent);
+
+				// Return version if found and it's a valid package.json
+				if (packageJson.version) {
+					return packageJson.version;
+				}
+			} catch (error) {
+				// Continue to next path if this one fails
+				continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)